scripts/qa_crowbarsetup.sh

```diff
@@ -2108,8 +2113,14 @@ function custom_configuration
         ;;
         rabbitmq)
             if [[ $hacloud = 1 ]] ; then
-                proposal_set_value rabbitmq default "['attributes']['rabbitmq']['ha']['storage']['mode']" "'drbd'"
-                proposal_set_value rabbitmq default "['attributes']['rabbitmq']['ha']['storage']['drbd']['size']" "$drbd_rabbitmq_size"
+                if [[ "$want_database_sql_engine" == "mysql" ]] ; then
```

Comment:
Ah, sorry, I misread it.
Anyway, I think both `=` and `==` are correct and we're using both in mkcloud code